### PR TITLE
[10.0][MIG] cmis web proxy alf

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Comment by lmignon** [comment]
@max3903 I fixed the build by travis. Can you rebase your branch on 10.0?

**Comment by max3903** [comment]
@lmignon Done!

File: cmis_web_proxy_alf/static/src/js/form_widgets.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ + *---------------------------------------------------------
+ +*/
+
+ odoo.define('cmis_web_proxy_alf.form_widgets', function( require) {
+"use strict";
+
+var core = require('web.core');
+var form_widgets = require('cmis_web.form_widgets');
+var Model = require('web.Model');
+var $ = require('$');
```
Comment (lmignon) [inline cmis_web_proxy_alf/static/src/js/form_widgets.js]: @max3903 This line is not working in Odoo10.  `+var $ = require('$');` It must be removed.

**Comment by lmignon** [comment]
@max3903 This one  could interest you 1432a5397b9e8961c1f36132bbfaebf1663ec036

**Pull request merged**